assets/js/script.js

```diff
@@ -0,0 +1,46 @@
+// TODO
+
+// function for time (unsure of desired outcome as of now)
+
+// function for quote fetch
+// NOT USING QUOTES.rest here....
+const getQuote = function() {
+    // send fetch request
+    fetch("https://type.fit/api/quotes")
+      //then if ok parse response
+      .then(function(response) {
+          if (response.ok) {
+            return response.json();
+          } else {
+              return;
+            }
+      }).then(function(data) {
+          displayQuote(data);
+      })
+}
+
+const displayQuote = function (dataArr) {
+    // find <p> by id
+    let quoteOfDayEl = $('#quoteOfTheDay');
```

Comment:
Minor suggestion. As a standard variables for jQuery objects are recommended the name to start with $ as in `let $quoteOfDayEl = ...` so that when you use it later you already know this is a jQuery object.